po/zh-Hant.po

```diff
@@ -2,14 +2,15 @@ msgid ""
 msgstr ""
 "Project-Id-Version: Comprehensive Rust 🦀\n"
 "POT-Creation-Date: \n"
-"PO-Revision-Date: 2023-05-26 15:20+0200\n"
-"Last-Translator: Martin Geisler <mgeisler@google.com>\n"
+"PO-Revision-Date: 2023-05-31 09:11+0800\n"
+"Last-Translator: Hank Chen <kuanhungchen@google.com>\n"
 "Language-Team: Chinese (Traditional) <zh-l10n@lists.linux.org.tw>\n"
+"Language: zh_TW\n"
```

Comment:
Is this change intended?  Is there a reference for why it's zh_TW instead of zh-Hant?

Reply:
No. The "Language" and "X-Generator" fields were somehow generated by the Poedit.
They are removed now.

Reply:
In some languages (e.g. ja, de) they include X-Generator while others (e.g. zh-Hans, el) don't. Not sure if we want to have it included.

Reply:
As far as I know, nothing in our pipeline uses the metadata here — we derive the URL directly from the name of the PO file. So you can change this as you like :slightly_smiling_face: